release/RELEASE.md

```diff
@@ -3,12 +3,17 @@
 1. Make a PR that bumps the version number in the `VERSION`
    file and adds a new header and label for the new version in
    `docs/source/support/release-notes.rst` (see previous releases as examples).
-   Release notes should be cut and pasted under the new header from `unreleased.rst`.
-   Each change outlined in `unreleased.rst` is preceded by the section to
+   Release notes additions can be retrieved from commits using the following command:
+
+     for sha in $(git log --format=%H <revision range>); do git show --format=%b $sha | awk '/^$/{next} /^CHANGELOG$/{flag=1; next} flag'; done
```

Comment:
My awk is pretty bad, judging frm @garyverhaegen-da’s comments multiple changelog paragraphs should work but I think it would be good to point that out here.

Reply:
Unfortunately it won't if you have commit message bodies in between two commits with changelog additions as in the following example.
```
This is the PR title (#42)

* This is the first commit

It has a body

* This is the second commit

CHANGELOG

- [Sandbox] More sand in the box

* This is the third commit

In this commit message body I wrote a couple of lines to tell people how much I love Italian wine. It deserves to be persisted.

* This is the fourth commit

CHANGELOG

- [Sandbox] Replace sand with block and box with chain
```
The output of the script would be:
```
- [Sandbox] More sand in the box
In this commit message body I wrote a couple of lines to tell people how much I love Italian wine. It deserves to be persisted.
- [Sandbox] Replace sand with block and box with chain
```